autoprecompiles/src/lib.rs

```diff
@@ -306,9 +306,15 @@ impl<'a, M, B: Clone> Clone for VmConfig<'a, M, B> {
     }
 }
 
-pub trait InstructionMachineHandler<T, I> {
+pub trait InstructionHandler<T, I> {
     /// Returns the AIR for the given opcode.
     fn get_instruction_air(&self, instruction: &I) -> Option<&SymbolicMachine<T>>;
+
```

Comment:
I added this to this trait, technically it could be a separate trait but I tried to keep it simple